packages/runtime/src/helpers/functions.ts

```diff
@@ -364,58 +369,154 @@ const changeExtension = (file: string, extension: string) => {
   return join(dirname(file), base + extension)
 }
 
-const getSSRDependencies = async (publish: string): Promise<string[]> => {
+const getPrerenderManifest = async (publish: string) => {
   const prerenderManifest: PrerenderManifest = await readJSON(join(publish, 'prerender-manifest.json'))
 
-  return [
-    ...Object.entries(prerenderManifest.routes).flatMap(([route, ssgRoute]) => {
-      if (ssgRoute.initialRevalidateSeconds === false) {
-        return []
-      }
+  return prerenderManifest
+}
+
+type CachedPageValue = Extract<ResponseCacheValue, { kind: 'PAGE' }> | { html: object | string }
+type PageCacheData = {
+  value: CachedPageValue
+  lastModified: number
+}
+
+const getPrerenderedBlobStoreContent = async (
+  prerenderManifest: PrerenderManifest,
+  publish: string,
+): Promise<Array<{ key: string; data: PageCacheData }>> => {
+  const limit = pLimit(Math.max(2, cpus().length))
```

Comment:
Going with the same logic used in https://github.com/netlify/next-runtime/blob/a30ffbace64ffb447c9a8c118cc1faac63cd8db9/packages/runtime/src/helpers/files.ts#L153